src/PythonBindings/VectorPyList.hpp

```diff
@@ -11,20 +11,39 @@
 
 namespace bp = boost::python;
 
+/*!
+ * \ingroup PythonBindingsGroup
+ *\brief A namespace containing functions for binding Python wrappers
+ *to their respective c++ classes and functions, as well as helper functions for
+ *these wrappers.
+ */
+
```

Comment:
We typically haven't been putting a blank line between the Doxygen comment and what it's documenting. Would you mind removing them in this PR?